ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-master.yaml

```diff
@@ -43,9 +43,9 @@ tests:
   steps:
     cluster_profile: aws
     test:
-    - as: bin-test
+    - as: critical-test
       cli: latest
-      commands: ./bin/extended-platform-tests run openshift/conformance/parallel --run "High-24829"
+      commands: ./bin/extended-platform-tests run all --dry-run|grep -E "OLM.*Critical"|./bin/extended-platform-tests run -f -
```

Comment:
@jianzhangbjz for later usage, please add 
```console
grep -v "VMonly"
```

I am not sure if we need to add 
```console
grep -v "Disruptive"
```
I also suggest to add it, but it is up to you.

Reply:
Thanks!  It's OK to run the `Disruptive` cases there.